system/CodeIgniter.php

```diff
@@ -400,6 +420,8 @@ private function isWeb(): bool
      *
      * @throws PageNotFoundException
      * @throws RedirectException
+     *
+     * @deprecated $returnResponse is deprecated, and no longer used.
```

Comment:
So what happens if a user called `run()` without returnResponse and `handleRequest()` with it? Is this a changed behavior?

Reply:
I can't think of any such use case, but I have fixed it so that the behavior does not change.